analyze-data/terra/interactive-analysis-with-jupyter-notebooks/code-library-for-biodata-catalyst.md

```diff
@@ -6,29 +6,29 @@ description: >-
 
 # Code Library for BioData Catalyst
 
-#### Author: Beth Sheets \(UC Santa Cruz, Genomics Institute\)
+#### Authors: Beth Sheets \(UC Santa Cruz, Genomics Institute\), Brian Hannafious \(UC Santa Cruz, Genomics Institute\)
 
-The [BioData Catalyst Collection workspace](https://terra.biodatacatalyst.nhlbi.nih.gov/#workspaces/biodata-catalyst/BioData%20Catalyst%20Collection) on Terra is a continually evolving and expanding source of example code that demonstrates tools relevant to conducting research in the ecosystem. The examples are primarily written as Jupyter notebooks that employ Python or R tooling. 
+The [BioData Catalyst Collection workspace](https://terra.biodatacatalyst.nhlbi.nih.gov/#workspaces/biodata-catalyst/BioData%20Catalyst%20Collection) on Terra is a continually evolving and expanding source of example code that demonstrates tools relevant to conducting research in the ecosystem. The examples are primarily written as Python and R Jupyter notebooks.
 
-Below, we highlight relevant tools that are introduced in the BioData Catalyst Collection. 
+Below, we highlight relevant tools introduced in the BioData Catalyst Collection. 
 
 ## Terra Data Table Tooling
 
 Before using this this tool, we suggest you review the documentation [Getting Started with Gen3 data in Terra](https://support.terra.bio/hc/en-us/articles/360038087312). 
 
-The Jupyter notebook [terra\_data\_table\_util](https://app.terra.bio/#workspaces/biodata-catalyst/BioData%20Catalyst%20Collection/notebooks/launch/terra_data_table_util.ipynb) provides functions that consolidate multiple Gen3 graph-structured data tables in Terra to create a single consolidated table that is easier to use. The content of the combined table produced is configurable.
+The Jupyter notebook [terra\_data\_table\_util](https://app.terra.bio/#workspaces/biodata-catalyst/BioData%20Catalyst%20Collection/notebooks/launch/terra_data_table_util.ipynb) provides functions that consolidate Gen3 graph-structured data into a single Terra data table. The content of the consolidated table is configurable.
 
-The default behavior is to produce a table keyed by subject id, with one row per subject, for all subjects in a Terra Workspace. This table may include the genomic data, harmonized clinical metadata, or both, along with the associated administrative information.
+The default behavior is to produce a table keyed by subject id, with one row per subject, for all subjects in a Gen3 graph. This table may include genomic data, harmonized clinical metadata, or both, along with associated administrative information.
```

Comment:
I think Terra workspace is correct here -- this is after users have handed off data from Gen3 to the workspace, and they want to combine data tables for all subjects handed off to that workspace.

Reply:
gotcha, will fix.